pmm/pmm-2-0-documentation-pdf.groovy

```diff
@@ -0,0 +1,61 @@
+pipeline {
+    agent {
+        label 'micro-amazon'
+    }
+    parameters {
+        string(
+            defaultValue: 'git@github.com:percona/pmm-doc',
+            description: 'Repository name for build',
+            name: 'REPO_NAME')
+        string(
+            defaultValue: 'master',
+            description: 'Tag/Branch for build',
+            name: 'BRANCH_NAME')
+    }
+    options {
+        buildDiscarder(logRotator(daysToKeepStr: '14', numToKeepStr: '5'))
+    }
+    triggers {
+        pollSCM 'H/15 * * * *'
+    }
+    stages {
+        stage('Checkout') {
+            steps {
+                deleteDir()
+                git poll: false, branch: BRANCH_NAME, url: REPO_NAME
+            }
+        }
+        stage('Publish') {
+            steps {
+                withCredentials([sshUserPrivateKey(credentialsId: 'aws-jenkins', keyFileVariable: 'KEY_PATH', passphraseVariable: '', usernameVariable: 'USER')]) {
+                    sh '''
+                        sudo yum -y install docker
+                        sudo usermod -aG docker ec2-user
+                        sudo service docker start
+                        echo BRANCH = \${BRANCH_NAME}
+                        echo "Building in: " `pwd`
+                        cd doc
+                        sudo docker pull perconalab/pmm-doc
```

Comment:
Maybe we can use sg docker -c "...."  to avoid using 'sudo' command...

Reply:
done please check again